src/api/v1/prerequisites/is_child_organisation.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * "organisation_details-child" route pre-requisite
+ *
+ * Determines if the authenticated user is trying to access an organisation
+ * which is actually a "child" entity.
+ *
+ * Conditions under which this is true for organisation X:
+ * - User is an admin for X
```

Comment:
Is this correct? It sound like you are describing the different permission flags here.

> User is an admin for X

This would be an `-own` flag
> User is an admin for a funding body which owns at least one funding programme of which X is a member

`-parent` flag

> User is a Twine admin

I designed the permissions hierarchy from the users perspective (as they are the ones accessing routes), rather than from an organisations perspective. 

`-parent` flag

Reply:
Yes, the descriptions are more or less associated to permission flags. We need these pre-reqs to actually verify the child/parent/own relations (as discussed in #87)

Are you just saying that the description should be different (i.e. reference flags explicitly)?

Reply:
Your descriptions are not what I imagined. 

My definitions are:
```
Twine Organisation X (Admin user A)
|
|__Funding Body Organisation Y (Admin user B)
  |
  |_ Community Business Organisation Z (Admin user C)
    |
    |_ _(Visitor user D)
```
#### - parent
- User D tries to access resources owned by Org Z
_eg_ a visitor adds a visit log to their CB
- User C tried to access resources owned by Org X
_eg a CB accesses details about their funding body_ 

#### - child
- User B tries to access resources owned by Org Z
_eg an admin user for a funding body gets details of all visits from the CB they fund_

#### - own
- User C tries to access resources owned by Org Z
_eg an admin user gets details of all visits at their CB_

Reply:
> > User is an admin for X

> This would be an `-own` flag

Yeah you're right

> > User is an admin for a funding body which owns at least one funding programme of which X is a member

> `-parent` flag

In your example this would be like User B accessing resources owned by Org Z, which you've said falls under "child"

> User is a Twine admin

In your example this is like User A accessing resources owned by Org Z, also "child".

So I think just my first example is wrong, the other two are OK?

Reply:
Yeah, you’re right.